[PLACES-336] Move access token from response body to set-cookie header and fix tests

aiyan opened the pull request:
This resolves [PLACES-336](https://pathcheck.atlassian.net/browse/PLACES-336?atlOrigin=eyJpIjoiNmFjYWFlZmY0ODg1NDZmMThlYjAyYjRlNWM0N2FkNzUiLCJwIjoiaiJ9).

Notable changes:
- JWT is now returned in a `Set-Cookie` header instead of in the response body.
- Login tests were modified to reflect the changes.
- All integration tests were modified to replace the `Authorization` header with the `Cookie` header that includes the access token.
- Pre-commit hook ran a code formatter, so other files were changed.

This requires the appropriate changes in the front end in order for the back end to integrate without issues.

krishnadurai commented on app/api/auth/controller.js:
```diff
@@ -22,11 +22,20 @@ exports.login = (req, res) => {
-      maps_api_key: process.env.SEED_MAPS_API_KEY,
-    });
+    res
+      .status(200)
```
Can we change the response to `204 HTTP Status Code` and drop the body?

aiyan commented:
@krishnadurai I removed the response body and changed the status code to 204.

Let's wait for the front end to make the appropriate changes in their access token handling before we merge.
